src/main/java/core/basesyntax/Colors.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax;
+
+public enum Colors {
```

Comment:
I think it is better to follow next code-style practices, using enums:
	1. Enum class name is single value: Color, State, Level, Size etc.
	2. Enum values should be capitalized as they are constants.
	3. (optional) Maybe it is better to sort it in alphabetic order, for readability, because colors bring same logic and meaning.